src/plugins/intel_gpu/src/kernel_selector/kernels/permute/permute_kernel_f_y_axes.cpp

```diff
@@ -0,0 +1,157 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "permute_kernel_f_y_axes.h"
+#include "kernel_selector_utils.h"
+#include "common_tools.h"
+#include <string>
+#include <functional>
+#include <cmath>
+
+namespace kernel_selector {
+
+static size_t GetDivisor(const size_t input_size) {
+    std::vector<size_t> v = {  /*32,*/ 16, 8, 4, /*2,*/ 1 };
+    auto is_devided = [input_size](int i){ return input_size%i == 0; };
+    auto result = std::find_if(begin(v), end(v), is_devided);
```

Comment:
What value will take tile size if input width (height) is not evenly divisible by higher then 1 values?

Reply:
for now, it is supported only cases where it is divisible by 4, 8 etc